### ISSUE TITLE
The game's 3.24.4.X update broke your plugins

### ISSUE THREAD

**Issue by CatzMR** [TavstalDev/TShop2#7]
The main problem at the moment seems to be the inability to purchase vehicles.
`[TShop] [EXCEPTION] > Error in HUDManager:
[TShop] [ERROR] > System.InvalidCastException: Specified cast is not valid.
  at Tavstal.TLibrary.Helpers.Unturned.UAssetHelper.FindVehicleAsset (System.UInt16 id) [0x00007] in <8538b98ef8c943e98c893ba24f10f5b4>:0
  at Tavstal.TShop.Managers.UIManager.UpdateProductPage (Rocket.Unturned.Player.UnturnedPlayer player) [0x0024a] in <6cca04b647834931ad0d3bfcaa3447b8>:0
[TShop] [WARNING] > Failed to get the asset name for 'vehicle' with 1 id. Exception:
[TShop] [EXCEPTION] > System.Exception: The asset is not a vehicle asset. Please review your database.
  at Tavstal.TShop.Compability.Product.GetName () [0x0004b] in <6cca04b647834931ad0d3bfcaa3447b8>:0
[TShop] [EXCEPTION] > Error in HUDManager:
[TShop] [ERROR] > System.InvalidCastException: Specified cast is not valid.
  at Tavstal.TLibrary.Helpers.Unturned.UAssetHelper.FindVehicleAsset (System.UInt16 id) [0x00007] in <8538b98ef8c943e98c893ba24f10f5b4>:0
  at Tavstal.TShop.Managers.UIManager.UpdateProductPage (Rocket.Unturned.Player.UnturnedPlayer player) [0x0024a] in <6cca04b647834931ad0d3bfcaa3447b8>:0`

**Comment by CatzMR** [comment]
No, no, no, the problem is not only with vehicle purchases, it is currently unable to perform any purchase operations.
```
"/buyv 1"
InvalidCastException: Specified cast is not valid.
  at Tavstal.TShop.CommandBuyVehicle.Execute (Rocket.API.IRocketPlayer caller, System.String[] args) [0x000b5] in <6cca04b647834931ad0d3bfcaa3447b8>:0
  at System.Runtime.CompilerServices.AsyncMethodBuilderCore+<>c.<ThrowAsync>b__7_0 (System.Object state) [0x00000] in <47fc8c70fa834cbf8141d7c1a7589125>:0
  at UnityEngine.UnitySynchronizationContext+WorkRequest.Invoke () [0x00002] in <fcaf289839f24125a059795f150fce14>:0
  at UnityEngine.UnitySynchronizationContext.Exec () [0x00056] in <fcaf289839f24125a059795f150fce14>:0
  at UnityEngine.UnitySynchronizationContext.ExecuteTasks () [0x00014] in <fcaf289839f24125a059795f150fce14>:0
```

**Comment by CatzMR** [comment]
When do you plan to fix it?

**Comment by TavstalDev** [comment]
The issue is not in TShop, I have moved the issue to TavstalDev/TLibrary#1
